.gitignore

```diff
@@ -1 +1,15 @@
-__*__
\ No newline at end of file
+
+__*__
+.vscode
+#logging info
+log*
+nohup.out
+# outputs 
+*output*
+*model_*
+# same as main train 
+main_continue.py
+# Jupyter checkpoint files
+*.ipynb*
+# unused scripts
+test*
```

Comment:
please add an extra new line character at the end of files. This will remove the red symbol!